WorkflowCombine/Sources/Logger.swift

```diff
@@ -14,8 +14,14 @@
  * limitations under the License.
  */
 
+import Foundation
 import os.signpost
 
+private let workerLoggingEnabled: Bool = {
+    let env = ProcessInfo.processInfo.environment
+    return env["com.squareup.workflow.workerLogging"] != nil
```

Comment:
Should this check for a bool, not just nil? Eg what if this is set to false?

Reply:
that would probably be a better check. i guess `(... as NSString).boolValue` is perhaps the way to go?

Reply:
I believe that's the standard way to check, yeah